From 659d91acdaf1822b3652a1111032df4b2bcf7b22 Mon Sep 17 00:00:00 2001
From: Loic Poulain <lpoulaix>
Date: Wed, 19 Dec 2012 14:01:56 +0100
Subject: [PATCH 62/78] Wi-Fi calibrator: Print errno content if NVS file
 creation fails.

BZ: 75644

This patch was created to monitor the root cause of NVS file creation failures.

Change-Id: I36c327526902f7489d64f7eedda0c92e8655af61
Signed-off-by: Loic Poulain <lpoulaix>
Reviewed-on: http://android.intel.com:8080/84515
Reviewed-by: Poulain, LoicX <loicx.poulain@intel.com>
Reviewed-by: De Cesco, Jonathan <jonathan.de.cesco@intel.com>
Reviewed-by: Garcia, JeremieX <jeremiex.garcia@intel.com>
Reviewed-by: Bachot, Jean-michel <jean-michel.bachot@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 calibrator/nvs.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/calibrator/nvs.c b/calibrator/nvs.c
index b55c3f5..bd1403f 100644
--- a/calibrator/nvs.c
+++ b/calibrator/nvs.c
@@ -705,7 +705,8 @@ int create_nvs_file(struct wl12xx_common *cmn)
 	new_nvs = open(cmn->nvs_name,
 		O_WRONLY | O_CREAT | O_TRUNC, S_IRUSR | S_IWUSR);
 	if (new_nvs < 0) {
-		fprintf(stderr, "%s> Unable to open new NVS file\n", __func__);
+		fprintf(stderr, "%s> Unable to open new NVS file"
+					"(%s)\n", __func__, strerror(errno));
 		return 1;
 	}
 
-- 
1.7.9

